Cargo.toml

```diff
@@ -48,18 +48,20 @@ std = []
 # requires the latest stable
 # this will have a tighter MSRV before stabilization
 kv_unstable = ["value-bag"]
-kv_unstable_sval = ["kv_unstable", "value-bag/sval", "sval"]
+kv_unstable_sval = ["kv_unstable", "value-bag/sval", "sval", "sval_ref"]
 kv_unstable_std = ["std", "kv_unstable", "value-bag/error"]
 kv_unstable_serde = ["kv_unstable_std", "value-bag/serde", "serde"]
 
 [dependencies]
 serde = { version = "1.0", optional = true, default-features = false }
-sval = { version = "=1.0.0-alpha.5", optional = true, default-features = false }
-value-bag = { version = "=1.0.0-alpha.9", optional = true, default-features = false }
+sval = { version = "2.1", optional = true, default-features = false }
```

Comment:
I ended up skipping a stable `1.x` for this library in the end, because the API changed fairly significantly between beta releases.